features/google/src/model/importers/utils/importer-utils.js

```diff
@@ -9,3 +9,7 @@ export function readCsvFromText(csvText) {
         headersEnum,
     };
 }
+
+export function removeTimezone(dateString) {
+    return dateString.split(" ").slice(0, -1).join(" ");
+}
```

Comment:
if we store Dates in UTC, with their timezone offet extra, we wouldn't need that.
In JS we can simply do, e.g.:
```e.g.
let theDate = new Date( Date.UTC(2022, 30, 5) );
console.log(theDate.toUTCString() );
```
The UTC string is formatted as YYYY-MM-DDZHH:mm:ss.sssZ.

If the Z is present, the Date is set to UTC. If the Z is not present, it's set to local time.

Otherwise, even in your solution, I feel this function is redundant. 
Doing above:
```
new Date(date.toISOString().slice(0, -1));
```
would create a date in format:
👉️ Mond May 30 2022 09:35:31 GMT+0300
without needing to call the `split` and thus this function.

(But, do we want this "GMT+0300" to be stored?)